docs/Getting started.md

```diff
@@ -41,12 +41,11 @@ Create, in the same directory as Sparksoniq, a file data.json and put the follow
 
 ## Running simple queries locally
 
-In a shell, from the directory where the sparksoniq .jar lies, type, all on one line:
+In a shell, from the directory where the rumble .jar lies, type, all on one line:
 
-    spark-submit --class sparksoniq.ShellStart --master local[*] --deploy-mode client
-                 sparksoniq-0.9.7.jar
+    spark-submit --master local[*] --deploy-mode client rumble-1.0.jar --shell yes
                  
-The Sparksoniq shell appears:
+The Rumble shell appears:
```

Comment:
I would recommend removing temporarily or updating the Sparksoniq banner shown below as well.

Reply:
Done